templates/leptos-ssr/content/src/app.rs

```diff
@@ -1,35 +1,57 @@
-use leptos::*;
+use components::{Route, Router, Routes};
+use leptos::{prelude::*, task::spawn_local};
 use leptos_meta::*;
 use leptos_router::*;
 
+#[cfg(feature = "ssr")]
+pub fn shell(options: LeptosOptions) -> impl IntoView {
+    view! {
+        <!DOCTYPE html>
+        <html lang="en">
+            <head>
+                <AutoReload options=options.clone() />
+                <HydrationScripts options=options.clone()/>
+                <MetaTags/>
+                <Stylesheet id="leptos" href="/pkg/{{project-name | snake_case}}.css"/>
+            </head>
+            <body>
+                <App/>
+            </body>
+        </html>
+    }
+}
+
 #[component]
 pub fn App() -> impl IntoView {
     // Provides context that manages stylesheets, titles, meta tags, etc.
     provide_meta_context();
 
+    let fallback = || view! { "Page not found." }.into_view();
+
     view! {
-      <Stylesheet id="leptos" href="/pkg/{{project-name | snake_case}}.css"/>
+        <Meta name="charset" content="UTF-8"/>
```

Comment:
Is there no css file in this one? It's fine if not, but if so it's removed it. It might be better to keep it

Reply:
I moved this to the [shell function](https://github.com/fermyon/leptos-spin/pull/48/files#diff-782be6a82ee08bd1b4914a1c0ca4dba43937f23e80b5d065f06287678ef51ee1R15) -- I recall that was the pattern I saw in some Leptos examples but I am certainly not a Leptos expert! Let me know if it is preferable to keep the stylesheet in App

Reply:
That's a good way to do it, I missed that